repobee_sanitizer/sanitizer.py

```diff
@@ -22,6 +22,13 @@
 PLUGIN_NAME = "sanitizer"
 LOGGER = daiquiri.getLogger(__file__)
 
+
+class SanitizeError(plug.PlugError):
+    def __init__(self, msg: str, cause: Optional[Exception] = None):
+        self.msg = msg
+        super().__init__(msg, cause)
```

Comment:
Put this somewhere else to avoid the circular import. Also, is this something that you want to be part of the "public API" of sanitizer (even though sanitizer isn't designed as a library)?